### PR TITLE
docs(aws): Adding documentation for AWS launch templates, mixed ASGs, and use case examples

### PR DESCRIPTION
**Pull request by pdk27**
## Summary
Adding documentation for AWS launch templates, mixed ASGs, and use case examples
Linking new docs under [Additional Features](https://spinnaker.io/docs/setup/other_config/features/)

Approved PR cut to old repo: https://github.com/spinnaker/spinnaker.github.io/pull/2187

### CODE REVIEW

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Here are some configuration parameters to consider:
+* Use MixedInstancesPolicy features detailed [here](/features/server-group-launch-settings/aws-ec2/launch-templates.md).
+  For example,
+   * Launch your Spot capacity from optimal instance pools using `spotAllocationStrategy` <b>caapcity-optimized</b>
+   * Specify a flexible set of instance types for your server group using `launch template overrides`.
+* Enable `capacityRebalance` to allow EC2 Auto Scaling to monitor and automatically respond to changes that affect availability of your Spot Instances. This feature works best with the `capacity-optimized` spotAllocationStrategy. 
+  For more information about capacity rebalancing, see [Amazon EC2 Auto Scaling Capacity Rebalancing.](https://docs.aws.amazon.com/autoscaling/ec2/userguide/capacity-rebalance.html)
+
+Learn more about [AWS recommended best practices.](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-best-practices.html).
```
Comment (brian-armory) [inline content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md:35]:
```suggestion
Learn more about [AWS recommended best practices](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-best-practices.html).
```

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title:  "AWS EC2 Server Group Launch Settings"
+linkTitle:  "AWS EC2 Server Group Launch Settings"
+weight: 1
```
Comment (brian-armory) [inline content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md:4]:
```suggestion
title:  "AWS EC2 Server Group Launch Settings"
linkTitle:  "AWS EC2 Server Group Launch Settings"
weight: 1
description: An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience with Spinnaker.
```

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+## Overview
+An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience with Spinnaker.
```
Comment (brian-armory) [inline content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md:8]:
```suggestion
```

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/launch-templates-setup.md [hunk 4]
```diff
@@ -1,31 +1,28 @@
 ---
-title:  "AWS Launch Templates"
-
+title:  "AWS EC2 Launch Templates Setup"
 ---
 
-> Please note that you should only proceed with this if you have [AWS EC2](/docs/setup/install/providers/aws/aws-ec2) configured as a cloud provider. These features requires version 1.24.
+{% include toc %}
```
Comment (brian-armory) [inline content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/launch-templates-setup.md:5]:
```suggestion
```

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/launch-templates.md [hunk 5]
```diff
@@ -0,0 +1,319 @@
+---
+
+title: "AWS EC2 Launch Templates"
+---
+
+{% include toc %}
```
Comment (brian-armory) [inline content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/launch-templates.md:6]:
```suggestion
```

**Review by brian-armory — CHANGES_REQUESTED** [review]
Thanks for moving this over! I had a few minor things (mostly due to how Hugo works vs Jekyll). Will merge once they're resolved

File: content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md (hunk 2 above)
Comment (pdk27) [reply content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md:4]: Is it okay for the description to be repeated under `Overview`?

**Review by brian-armory — APPROVED** [review]
👍🏼

**Pull request merged**